docs/reference/cli.md

```diff
@@ -705,6 +705,10 @@ $ nextflow lineage SUBCOMMAND [arg ..]
 
 The `lineage` command is used to inspect lineage metadata. Data lineage can be enabled by setting `lineage.enabled` to `true` in your Nextflow configuration (see the {ref}`config-lineage` config scope for details).
 
+:::{tip}
+Set `lineage.enabled` in your `~/.nextflow/config` file to enable it globally.
```

Comment:
This is not useful unless providing a common lineage path

Reply:
The way I struggled with this was that I added `lineage.enabled` to a pipeline config, and then ran that pipeline in a different location. It correctly used lineage and generated a `.lineage` directory, but when I tried to do anything with it I couldn't, because the `nextflow lineage` commands couldn't find a config to enable them.

@jorgee mentioned maybe making these commands specifically default to having lineage enabled, which would solve it.

I do wonder if it's still useful though. Globally enabling it means that lineage is always created, wherever you run. Otherwise you need to manually enable for each run, and when you want to use the lineage (eg. to debug failed resume) you can't. So I find this useful even without a common lineage path.

Reply:
Moving this tip to the guide